Resources/Prototypes/Catalog/Fills/Crates/engineering.yml

```
@@ -215,3 +215,25 @@
+    - id: RPD
+
+# Assmos - Extinguisher Nozzle
+- type: entity
+  id: CrateEngineeringFirefighterBackpackTank
+  parent: CrateEngineering
+  name: firefighter crate
+  description: Contains a firefighter backpack water tank.
+  components:
+  - type: StorageFill
+    contents:
+      - id: ClothingBackpackFirefighterTank
```

yamllint (1.35.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)